recipes/nnetsauce/meta.yml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nnetsauce" %}
+{% set version = "0.8.3" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/Techtonique/{{ name }}/tree/master/dist/{{ name }}-{{ version }}.tar.gz
+    sha256: 2163e76d6edc6339207b8ee89275ab0fc9c8846f1f6a6997c9fed5628ad5ae0e
+
+build:  
+    number: 0  
+    script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+    build:    
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+    host:
+        - python
+        - pip
+        - setuptools 
+        - cython
+    run:
+        - python
+        - numpy
+        - scipy
+        - scikit-learn
+        - joblib
+        - threadpoolctl
+
+test:  
+  imports:
+    - python
+    - numpy
+    - scipy
+    - scikit-learn
+    - joblib
+    - threadpoolctl
+    - nose2
+    - coverage
+
+about:
+  home: https://techtonique.github.io/nnetsauce/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Randomized and quasi-randomized networks'
+
+description: |
```

Comment:
The `description`, `doc_url` and `dev_url` should be indented so that they are in the `about` section.

Reply:
fixed